docs/source/index.md

```diff
@@ -1,13 +1,13 @@
 ---
 title: Introduction
-description: A strongly-typed, caching GraphQL client for Android and Kotlin multiplatform, written in Kotlin
+description: A strongly-typed, caching GraphQL client for the JVM
```

Comment:
Apollo Android isn't limited to the JVM. It was already the case with 2.x and even more so with 3.x where the core runtime and cache has been rewritten in Kotlin and now supports iOS/MacOS in addition to Android.

Reply:
Also, I don't think it's important to specify "the JVM". Java developers know that Kotlin libraries can be used in the JVM at this point, the language has been around and "advertised" for over 5 years now, I don't think we need to wait another 5.

Reply:
I don't know if it's important to specify "for Android" either, it's a given for a project named "apollo-android".

That now makes me think it'd be better named "apollo-kotlin". That would be less confusing regarding what it actually is and how compatible it is.